### ISSUE TITLE
[BUG][PYTHON] ApiTypeError when receiving a json response body with additionalProperties

### ISSUE THREAD

**Issue by Straktisie** [OpenAPITools/openapi-generator#9808]
##### Description

Hello,
when receiving the response for a rest call containing additionalProperties (see example below) the swagger-client crashes with following error message:

>   File "/home/johannes/PycharmProjects/openapi_client/main.py", line 38, in <module>
    api_response = api_instance_hubserver.hubserver_props_get(fetch=['HTTP_Port'])
  File "/home/johannes/PycharmProjects/openapi_client/openapi_client/api_client.py", line 772, in __call__
    return self.callable(self, *args, **kwargs)
  File "/home/johannes/PycharmProjects/openapi_client/openapi_client/api/hub_server_interface_api.py", line 325, in __hubserver_props_get
    return self.call_with_http_info(**kwargs)
  File "/home/johannes/PycharmProjects/openapi_client/openapi_client/api_client.py", line 834, in call_with_http_info
    return self.api_client.call_api(
  File "/home/johannes/PycharmProjects/openapi_client/openapi_client/api_client.py", line 409, in call_api
    return self.__call_api(resource_path, method,
  File "/home/johannes/PycharmProjects/openapi_client/openapi_client/api_client.py", line 224, in __call_api
    return_data = self.deserialize(
  File "/home/johannes/PycharmProjects/openapi_client/openapi_client/api_client.py", line 325, in deserialize
    deserialized_data = validate_and_convert_types(
  File "/home/johannes/PycharmProjects/openapi_client/openapi_client/model_utils.py", line 1609, in validate_and_convert_types
    input_value[inner_key] = validate_and_convert_types(
  File "/home/johannes/PycharmProjects/openapi_client/openapi_client/model_utils.py", line 1541, in validate_and_convert_types
    converted_instance = attempt_convert_item(
  File "/home/johannes/PycharmProjects/openapi_client/openapi_client/model_utils.py", line 1423, in attempt_convert_item
    raise get_type_error(input_value, path_to_item, valid_classes,
openapi_client.exceptions.ApiTypeError: Invalid type for variable 'HTTP_Port'. Required value type is dict and passed type was str at ['received_data']['HTTP_Port']

##### openapi-generator version

5.2.0-SNAPSHOT

##### OpenAPI declaration file content or url

```yaml
  /api/v0/hubserver/props:
    get:
      operationId: hubserver_props_get
      responses:
        '200':
          description: OK
          content:
            application/json:
              schema:
                title: hubserver_props_get_resp
                type: object
                description: prop
                additionalProperties:
                  oneOf:
                    - type: integer
                    - type: string
                    - type: object
```

##### Generation Details

used docker container to create python client

**Comment by spacether** [comment]
Our generator does not support inlined composed (oneOf) schemas.
Can you move your:
```
                  oneOf:
                    - type: integer
                    - type: string
                    - type: object
```
schema into its own component and then use a $ref to the component in your response definition?

**Comment by spacether** [comment]
This was fixed in python 6.2.0 which supports inline schemas of any depth
